conanfile.py

```diff
@@ -73,11 +73,9 @@ def requirements(self):
         self.requires("grpc/1.27.3@{}".format(self._orbit_channel))
         self.requires("c-ares/1.15.0", override=True)
         self.requires("llvm-core/12.0.0@{}".format(self._orbit_channel))
-        self.requires("lzma_sdk/19.00@orbitdeps/stable#a7bc173325d7463a0757dee5b08bf7fd", override=True)
```

Comment:
lzma_sdk is a dependency of libprotobuf-mutator. Since it is now build as part of Orbit, it is no longer an override dependency, but a real dependency. (That means conan will generate cmake config files which are then consumed by CMake when building libprotobuf-mutator.